package.json

```diff
@@ -32,7 +32,8 @@
   "license": "MIT",
   "dependencies": {
     "findup-sync": "^0.2.1",
```

Comment:
It's still used to find the package.json itself, unfortunately. Tried to work out a way to use a single module, but doesn't seem to be possible.